### PR TITLE
Inline SVG for arrow in native select

### PR DESCRIPTION
**Pull request by seangates**
## Description
These changes update the markup of the select. It now uses inline SVG instead of the background SVG.

## Context
We should use inline SVG where possible, as it makes things more flexible.

## References
Related to [ebayui-core #63](https://github.com/eBay/ebayui-core/issues/63)

### CODE REVIEW

**Comment by senthilp** [comment]
@seangates What is the reasoning for this? Is it because we need to change the color of the icon? or is it for performance reasons? i.e. we are avoiding the CSS classes from being downloaded?

**Comment by seangates** [comment]
@senthilp It's all for consistency. We are trying to keep as much flexibility in our components as possible. Also, it's for performance, since we don't have to carry the additional icon if we use `ebay-icon` stamped on the page.

File: docs/_includes/ds6/select.html [hunk 1]
```diff
@@ -11,6 +11,9 @@ <h3>Default select</h3>
                     <option value="item2">Option 2</option>
                     <option value="item3">Option 3</option>
                 </select>
+                <svg class="icon icon--chevron-down-bold" focusable="false" height="24" width="24">
```
Comment (senthilp) [inline docs/_includes/ds6/select.html]: @seangates as we discussed, please add the documentation to mention that SVG definitions should be included, in order for this to work.

**Comment by ianmcburnie** [comment]
A few comments.

1. I'm not sure why we are doing this now, as this is a breaking change and wasn't on roadmap. Do we have to support other colours for the icon right now? Or can teams override in the interim? I'm also a little concerned that teams are overriding the color specified in the design system so soon. I thought everyone had decided on blue (or was it black!? lol).
2. Adding support for inline SVG is good, we should absolutely support it at some point, but background SVG should continue to be the default use case for Skin if possible (like we do in other modules), because it is the easiest method for those not using eBayUI (i.e. no stamping of svg required). If we go forward continuing to support background svg as the default case, we can probably do this enhancement in a backwards compatible way and then everyone is happy.
3.  I don't think we need to make the end user specify `icon icon--chevron-down-bold`. It's always a chevron down, so `select__icon` should be sufficient and we abstract that detail away. Again, this has been the pattern up to now and so I'm not sure we should go changing things.

**Comment by seangates** [comment]
Closed per comments.

The only part of this PR that is a necessity is the `select.browser.json`.

**Pull request closed without merging**